core/ide/src/main/java/org/alice/ide/cascade/ExpressionCascadeManager.java

```diff
@@ -163,6 +163,10 @@ public ExpressionCascadeContext popContext() {
   }
 
   public ExpressionCascadeContext popAndCheckContext(ExpressionCascadeContext expectedContext) {
+    if (this.contextStack.size() == 0) {
```

Comment:
What is the case this is handling?
I can't seem to get in a state with an empty stack.